templates/default/postfix-main.cf.erb

```diff
@@ -18,20 +11,21 @@ append_dot_mydomain = no
 readme_directory = no
 
 # TLS parameters
-smtpd_tls_cert_file=/etc/ssl/certs/ssl-cert-snakeoil.pem
-smtpd_tls_key_file=/etc/ssl/private/ssl-cert-snakeoil.key
+smtpd_tls_cert_file=<%= @postfix_cert_path %>
```

Comment:
Pointing to the cert/key file created by the recipe. You only need `smtpd_tls_cert_file` if the key and cert are in the same file.